frameworks/elastic/tests/test_sanity.py

```diff
@@ -167,6 +167,8 @@ def test_custom_yaml_base64():
 
 @pytest.mark.sanity
 @pytest.mark.timeout(60 * 60)
+@pytest.mark.skipif(sdk_utils.is_master_channel(),
```

Comment:
How about `sdk_utils.dcos_version_at_least('1.12')` to prevent changes to `sdk_utils` and rebuilds of all frameworks?